index.js

```diff
@@ -0,0 +1,4 @@
+export { buildJS, watchJS } from './lib/rollup.js';
+export { buildAndRunTests } from './lib/tests.js';
+export { compileSass } from './lib/sass.js';
```

Comment:
`compileSass` is taken from `scripts/gulp/create-style-bundle.js` in the frontend-shared repo with minimal modifications.